workloads/router-perf-v2/common.sh

```diff
@@ -24,13 +24,13 @@ get_scenario(){
   if [[ ${NUM_NODES} -ge ${LARGE_SCALE_THRESHOLD} ]]; then
     log "Large scale scenario detected: #workers >= ${LARGE_SCALE_THRESHOLD}"
     export NUMBER_OF_ROUTES=${LARGE_SCALE_ROUTES:-500}
-    CLIENTS=${LARGE_SCALE_CLIENTS:-"1 20 80"}
-    CLIENTS_MIX=${LARGE_SCALE_CLIENTS_MIX:-"1 10 20"}
+    CLIENTS=${LARGE_SCALE_CLIENTS:-"1 80"}
```

Comment:
cc: @jtaleric @dry923, changing default number of clients, as per 4.12 maxconns has been upgraded from 20K to 50K. Some context -> https://github.com/cloud-bulldozer/e2e-benchmarking/issues/519#issuecomment-1331964622